plugins/themes/default/styles/objects/article_summary.less

```diff
@@ -58,10 +58,12 @@
 	.galleys_links {
 		&:extend(.pkp_unstyled_list);
 		margin-top: @base;
+		display: flex;
+		flex-flow: row nowrap;
+		justify-content: start;
 
 		li {
-			display: inline-block;
-			margin-right: 1em;
+			margin-inline-end: 1.25rem;
```

Comment:
This should be `0.25rem`, not `1.25rem`.

Reply:
I've adjusted the margin due to the fact when I had changed to flexbox the parent container, the `LI `elements decreased the space between the buttons.

See the current distance (non flexbox): 
![image](https://user-images.githubusercontent.com/4992902/190150393-ec37fee4-c78d-4427-a7fc-c98ffd60fcae.png)

If I keepet `0.25rem` (with flexbox)
![image](https://user-images.githubusercontent.com/4992902/190150551-354fd8f5-de6a-4cc2-9f74-ff735ee58822.png)

And using `1.25rem` (with flexbox)
![image](https://user-images.githubusercontent.com/4992902/190150689-b5291541-10af-44cb-911a-528f8493d9bb.png)

I thought that this distance should be preserved for keeping enough room between elements and presenting easier user interaction (clicking) area.  Should I change back to the original value?

Reply:
Oh my mistake. I thought that the previous margin was `0.25rem` but it was `1em`. I'd suggest leaving it the way that it was, `1em`. Sorry that was my mistake.

Reply:
No worries!
I've made all adjustments. Could you please review it and let me know if any other is required to merge?